sync/handler_hello_test.go

```diff
@@ -20,12 +20,12 @@ func TestParsingHelloMessages(t *testing.T) {
 
 	t.Run("Receiving Hello message from a peer. Peer ID is not same as initiator.",
 		func(t *testing.T) {
-			signer := td.RandSigner()
+			valKey := td.RandValKey()
 			pid := td.RandPeerID()
 			initiator := td.RandPeerID()
 			msg := message.NewHelloMessage(pid, "bad-genesis", 0, 0,
 				td.state.LastBlockHash(), td.state.Genesis().Hash())
-			msg.Sign(signer)
+			msg.Sign([]*bls.ValidatorKey{valKey})
```

Comment:
This function doesn't perform any actions. It can be removed.
A better approach is to check the type of the error.